Content.Shared/Nutrition/Components/SatiationComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Shared.Nutrition.EntitySystems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Nutrition.Components;
+
+[Access(typeof(SatiationSystem))]
+[RegisterComponent, NetworkedComponent]
+[AutoGenerateComponentState, AutoGenerateComponentPause]
+public sealed partial class SatiationComponent : Component
+{
+    [DataField("hunger"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadWrite)]
```
DataField names are autogenerated now based on the field name.
(Do this for all DataFields not just this one)

Reply:
Explicit field names have been removed.